src/table.tsx

```diff
@@ -75,83 +127,207 @@ export const Table = React.forwardRef(
-
-                                        <Text2
-                                            as="div"
-                                            weight={idx === 0 && collapsedRowsMode ? 'medium' : 'regular'}
-                                            wordBreak={false}
+                    {content.length > 0 ? (
+                        content.map((row, rowIdx) => {
+                            const rowCells = !Array.isArray(row) ? row.cells : row;
+                            const rowActionsList = !Array.isArray(row) ? row.actions ?? [] : [];
+
+                            const actions = (
+                                <Inline space={16}>
+                                    {rowActionsList.map((action, index) => {
+                                        if ('Icon' in action || 'checkedProps' in action) {
+                                            // action is a CellAction object
+                                            return <CellActionIconButton key={index} {...action} />;
+                                        }
+                                        // action is a React.ReactElement
+                                        return action;
+                                    })}
+                                </Inline>
+                            );
+
+                            // buttons + inline space
+                            const actionsElementWidth = rowActionsList.length
+                                ? `calc(${iconContainerSize.small} * ${rowActionsList.length} + 16px * ${
+                                      rowActionsList.length - 1
+                                  })`
+                                : '0px';
+
+                            return (
+                                <tr key={rowIdx} style={{position: 'relative'}}>
+                                    {rowCells.map((cell, idx) => (
+                                        <td
+                                            key={idx}
+                                            className={classNames(styles.verticalAlign[rowVerticalAlign], {
+                                                [styles.rowFirstItem]: idx === 0,
+                                                [styles.rowLastItem]:
+                                                    idx === rowCells.length - 1 && !hasActionsColumn,
+                                                [styles.rowLastCollapsedItem]:
+                                                    idx === rowCells.length - 1 && collapsedRowsMode,
+                                            })}
+                                            style={{
+                                                // add space between top actions and content
+                                                marginRight:
+                                                    collapsedRowsMode && rowActionsList.length
+                                                        ? `calc(${actionsElementWidth} + 8px)`
```

Comment:
are nested `calc()` valid css? :/ (note `actionsElementWith` can have a `calc()`)

Reply:
Yes, it's valid (https://developer.mozilla.org/en-US/docs/Web/CSS/calc#rules_and_best_practices_while_using_calc):

![image](https://github.com/Telefonica/mistica-web/assets/25785151/247dfc1b-a53b-4d7d-b16e-ccad2d924486)

We actually use this nesting in many components, like in IconButton or Input Fields :)